tests/Gedmo/SoftDeleteable/SoftDeleteableDocumentTest.php

```diff
@@ -27,8 +30,8 @@ class SoftDeleteableDocumentTest extends BaseTestCaseMongoODM
     public const MODULE_CLASS = 'Gedmo\Tests\SoftDeleteable\Fixture\Document\Module';
```

Comment:
IMO, these constants that were not replaced can be safely removed, as these classes don't exist.

Reply:
By now, I'll merge this as is. Later, we can create a new PR for these removals.